src/qt/walletcontroller.cpp

```diff
@@ -86,6 +86,23 @@ void WalletController::closeWallet(WalletModel* wallet_model, QWidget* parent)
     removeAndDeleteWallet(wallet_model);
 }
 
+void WalletController::closeAllWallets(QWidget* parent)
+{
+    QMessageBox::StandardButton button = QMessageBox::question(parent, tr("Close all wallets"),
```

Comment:
`QMessageBox::question()` is [obsolete](https://doc.qt.io/qt-5.9/qmessagebox-obsolete.html#question-1). Could it be avoided?